govtool/frontend/src/components/organisms/DashboardCards/DirectVoterDashboardCard.tsx

```diff
@@ -127,7 +127,9 @@ export const DirectVoterDashboardCard = ({
       description: (
         <Trans
           i18nKey="dashboard.cards.directVoter.registerDescription"
-          values={{ votingPower: ada }}
```

Comment:
@j-dyczka 

Im not sure why we were passing voting power through here

Reply:
Must have been a bug